apiserver/facades/agent/uniter/uniter.go

```diff
@@ -68,10 +68,16 @@ type UniterAPI struct {
 	cloudSpec       cloudspec.CloudSpecAPI
 }
 
-// UniterAPI implements the latest version (v11) of the Uniter API,
+// UniterAPIV11 implements the latest version (v12) of the Uniter API,
+// with RemoveUpgradeCharmProfileData as NotSupproted.
```

Comment:
WatchUnitLXDProfileUpgradeNotifications and WatchLXDProfileUpgradeNotifications should be on this list too afaik.

Reply:
Removing LXDProfileAPI does this.